### PR TITLE
docs: update doc

### PR DESCRIPTION
**Pull request by Pavloro**

### CODE REVIEW

File: example/values-secrets.yaml
Comment (AlexanderBrevig) [inline example/values-secrets.yaml:1]: I think maybe we should add some top level description to README as well? And make the docs in values.yaml clearer so it get propagated to the chart documentation. What do you think?

**Review by AlexanderBrevig — CHANGES_REQUESTED** [review]
Moar cowbell (and more docs) 📜 ?

File: example/values-secrets.yaml
Comment (Pavloro) [reply example/values-secrets.yaml:1]: I updated the documentation in Confluence. We can also add a README file in valuse.yaml, but then we need to update both files when changes come.

File: .circleci/config.yml
Comment (AlexanderBrevig) [inline .circleci/config.yml:1]: move to GHA

**Review by AlexanderBrevig — APPROVED** [review]

**Review by AlexanderBrevig — APPROVED** [review]

**Comment by Pavloro** [comment]
LGTM :+1:

**Review by Glenn-Terjesen — APPROVED** [review]

**Pull request merged**